### DISCUSSION TITLE
Suggest all normal words in the tex file?

### DISCUSSION THREAD

**Discussion by ScyllaHide** [texstudio-org/texstudio#3854] [Q&A]
Hello,

i have a different simple editor here - AkelPad, which lets me autocomplete all possible words in the document. For texstudio this does only work for tex commands, now is there a similar feature, which lets you do that in texstudio? It would save me a lot typing for longer words. I mean tex commands would be triggered by \blabla - by the "\" anyway.

I also tried to google for answer and there was not really a result, so i thought i ask here.

Regards,
SH

**Comment by octaeder** [comment]
After typing part of the word simply use `ctrl+space`. In the completer panel you can use `shift+space` to switch categories:

![image](https://github.com/user-attachments/assets/9042319b-a6cf-4fb5-aa4c-98dc0e4c1ded)

**Comment by ScyllaHide** [comment]
Thanks for replying!

okay, but thats super annoying live texing during a lecture.

is it not possible to use the backslash as an indication that a latex commend is following and therefore no normal word is suggested?
if that is the solution, i can simply type the word and be faster than changing the ctrl+space.

i will have to try it. any idea, which version this was added?
because im still on TeXstudio 2.12.22 (git 2.12.22) -  which is from around 2020 i think.

**Reply by sunderme** [reply]
I would recommend to take a look at the user manual https://texstudio-org.github.io/editing.html#auto-completion

**Comment by fengzyf** [comment]
Another solution is to install an English input method that supports auto-suggestions and spell check features. For example: [hallelujahIM](https://github.com/dongyuwei/hallelujahIM/blob/master/README-En.md).
